### PR TITLE
Add sbt-updates as a default plugin to projects

### PR DESCRIPTION
**Pull request by adamdecaf**
You can now run the `depenedencyUpdates` sbt command and get output like the following.

```
[info] Found 15 dependency updates for <project-name>
[info]   ch.qos.logback:logback-classic      : 1.0.13           -> 1.1.3
[info]   ch.qos.logback:logback-core         : 1.0.13           -> 1.1.3
[info]   com.typesafe.akka:akka-actor        : 2.3.7  -> 2.3.12
[info]   com.typesafe.akka:akka-remote       : 2.3.7  -> 2.3.12
[info]   com.typesafe.akka:akka-slf4j        : 2.3.7  -> 2.3.12
[info]   com.typesafe.akka:akka-testkit:test : 2.3.7  -> 2.3.12
[info]   javax.servlet:javax.servlet-api     : 3.0.1            -> 3.1.0
[info]   joda-time:joda-time                 : 2.4              -> 2.8.1
[info]   junit:junit:test                    : 4.11             -> 4.12
[info]   org.joda:joda-convert               : 1.6              -> 1.7
[info]   org.scala-lang:scala-library        : 2.10.5           -> 2.11.7
[info]   org.slf4j:jcl-over-slf4j            : 1.7.5  -> 1.7.12
[info]   org.slf4j:log4j-over-slf4j          : 1.7.5  -> 1.7.12
[info]   org.slf4j:slf4j-api                 : 1.7.5  -> 1.7.12
[info]   org.specs2:specs2:test              : 2.4.2  -> 2.4.17           -> 3.3.1
[success] Total time: 16 s, completed Aug 5, 2015 8:30:50 AM
```

### CODE REVIEW

**Comment by adamdecaf** [comment]
The other neat thing is that setting `dependencyUpdatesFailBuild` can make our CI builds fail if we're out of date.

File: build.sbt [hunk 1]
```diff
@@ -75,3 +77,5 @@ publishArtifact in (Compile, packageDoc) := false
 credentials += Credentials(Path.userHome / ".ivy2" / ".banno_credentials")
 
 publishMavenStyle := true
+
+dependencyUpdatesExclusions := moduleFilter(organization = "com.banno")
```
Comment (rubbish) [inline build.sbt]: Should we put this in the default settings as well?

**Comment by rubbish** [comment]
:+1:

File: build.sbt (hunk 1 above)
Comment (adamdecaf) [inline build.sbt]: Oh, yea. We'll need to do that.

**Comment by jwinder** [comment]
:100:

**Comment by rubbish** [comment]
Also, I'm not totally sure or not, but do we have to do anything for the auto-plugin to activate? I'm not sure if all of our projects are yet using banno-sbt-plugin as an AutoPlugin, but I thought there was something we needed to do to make this sbt-updates activate.

**Comment by adamdecaf** [comment]
@rubbish I've seen some projects were still on version 5 (see https://github.com/Banno/sentry/commit/0443d376c1c1f933feff719c8b7ff8cd15f8b2d0)

**Pull request merged**